src/pages/userInvitation/mocks/pageInitConfig.js

```diff
@@ -62,7 +62,7 @@ export default {
 			name: 'Search User',
 			stepLabel: '{$step} - Search User',
 			description:
-				'Search for the user using their email address, username or ORCID iD. Enter at least one details to get started. If user does not exist, ypu can invite them to take up roles and be a part of your journal. If the user already exist in the system, you can view user information and invite to take a additional roles.',
+				'If user does not exist, ypu can invite them to take up roles and be a part of your journal. If the user already exist in the system, you can view user information and invite to take a additional roles.',
```

Comment:
Sorry I missed this on my first review. There's a small typo here and message could be improved:
```suggestion
				'If the user does not exist, you can invite them to take on roles and become part of your journal. If the user already exists in the system, you can view their information and invite them to take on additional roles.',
```